__mocks__/browser-request.js

```diff
@@ -1,16 +1,17 @@
 const en = require("../src/i18n/strings/en_EN");
 
 module.exports = jest.fn((opts, cb) => {
-    if (opts.url.endsWith("languages.json")) {
+    const url = opts.url || opts.uri;
+    if (url && url.endsWith("languages.json")) {
         cb(undefined, {status: 200}, JSON.stringify({
             "en": {
                 "fileName": "en_EN.json",
                 "label": "English",
             },
         }));
-    } else if (opts.url.endsWith("en_EN.json")) {
+    } else if (url && url.endsWith("en_EN.json")) {
         cb(undefined, {status: 200}, JSON.stringify(en));
     } else {
-        cb(undefined, {status: 404}, "");
+        cb(true, {status: 404}, "");
```

Comment:
I don't fully understand this change, but it's probably fine.

Reply:
The first callback arg is for the possible error, and we expect to have a truthy error value when there's a network failure like 404.